contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -51,18 +51,22 @@
         }"
       >
         <VLayout row class="px-3">
-          <IconButton
-            icon="collapseAll"
-            :text="$tr('collapseAllButton')"
-            @click="collapseAll"
-          />
+          <KRouterLink>
+            <KIcon
+              icon="collapseAll"
+              :text="$tr('collapseAllButton')"
+              @click="collapseAll"
+            />
+          </KRouterLink>
```

Comment:
I think that both of the uses of `KRouterLink` here can be replaced w/ `KIconButton` which will emit the `@click` event when clicked _and_ has the `icon` props.

One change here is that rather than the `:text` prop, you'll want to use the `:tooltip` prop like you've done in the other changes.

The `KRouterLink` is only necessary if we're looking to navigate the page as it is basically a [wrapper](https://github.com/learningequality/kolibri-design-system/blob/d26b3a42cf7ef92c95133db5b9e86162e344acf8/lib/buttons-and-links/KRouterLink.vue#L8) for the Vue `router-link` component.

Reply:
Just realized that @LianaHarris360 left similar feedback, sorry for the redundancy 😅